### PR TITLE
Additions and fixes.

### PR DESCRIPTION
**Pull request by gauravsitlani**

### CODE REVIEW

File: file_and_modules.md [hunk 1]
```diff
@@ -1,10 +1,9 @@
-## File operations
-* file is a storage format where we store information on the computer.
-* It might text files like music, video, code and other stuff or binary
-  files which are only understood able by computer.
+## Filesystem operations
+* A file is stored as a resource on the computer on top of a [filesystem](https://www.tldp.org/LDP/sag/html/filesystems.html) which is a storage format.
+* Files can be text , music or media or any binaries which can only be understood by computer.
```
Comment (chkumar246) [inline file_and_modules.md]: There are two types of files, one which is understandable by user and by computer by like this. Add like this.

File: file_and_modules.md [hunk 2]
```diff
@@ -36,11 +35,11 @@
 ```
 
 ## Python Modules
-* Python provides a rich support of modules.
-* Modules/library are a collection of python scripts which can be reusable for writting complex program.
-* We can import a module using import keyword.
+* Python provides a rich support for modules.
+* Modules/libraries are a collection of python scripts which can be reusable for writing complex programs.
```
Comment (chkumar246) [inline file_and_modules.md]: add aka python packages.

**Review by chkumar246 — COMMENTED** [review]
Thanks for putting the patch, really appreciated.

File: file_and_modules.md (hunk 1 above)
Comment (gauravsitlani) [reply file_and_modules.md]: done!

File: file_and_modules.md (hunk 2 above)
Comment (gauravsitlani) [reply file_and_modules.md]: done!

**Pull request merged**